OpenRA.Mods.Common/Traits/Captures.cs

```diff
@@ -130,7 +130,7 @@ public override bool CanTargetFrozenActor(Actor self, FrozenActor target, Target
 					return false;
 				}
 
-				var health = target.Info.TraitInfoOrDefault<HealthInfo>();
+				var health = target.Actor.TraitOrDefault<IHealth>();
```

Comment:
I expect this will introduce crashes for the same reason as above.  Keep this querying `target.Info`.

Reply:
See my above comment.